callback_test.go

```diff
@@ -0,0 +1,50 @@
+//go:build manual_test
```

Comment:
Isn't it possible to remove the build tag by compiling the C program on the fly (`os/exec`)?

Reply:
Done